clients/wallet/src/stellar_wallet.rs

```diff
@@ -99,6 +99,14 @@ impl StellarWallet {
 		})?;
 
 		let cache = WalletStateStorage::new(cache_path, &pub_key, is_public_network);
+		// using a builder to decrease idle connections
+		// https://users.rust-lang.org/t/reqwest-http-client-fails-when-too-much-concurrency/55644/2
+		let client = reqwest::Client::builder()
+			// default is 90 seconds.
+			.pool_idle_timeout(Some(Duration::from_secs(60)))
+			// default is usize max.
+			.pool_max_idle_per_host(usize::MAX / 2)
+			.build()?;
```

Comment:
Did you notice a meaningful difference with this change?

Reply:
To be completely honest, I am unsure. 

However we encountered idle connections before and even talked about using `ClientBuilder`. I guess now's the right time.

I set the pool to half of default because [zero](https://users.rust-lang.org/t/reqwest-http-client-fails-when-too-much-concurrency/55644/5) is too huge a leap.